proto/proto-backplane-grpc/src/main/proto/deephaven/proto/session.proto

```diff
@@ -99,12 +107,12 @@ message HandshakeRequest {
    * - protocol = 0: most recent HandshakeResponse payload
    * - protocol = 1: payload is BasicAuth
    */
-  sint32 auth_protocol = 1;
+  sint32 auth_protocol = 1 [deprecated=true];
```

Comment:
Is it possible to deprecate the whole HandshakeRequest object, or no?

Reply:
It appears not. I used the answer at https://stackoverflow.com/a/43380742/860630 to guide this change, and spam the "this is deprecated, plan on us removing it!" notices.